collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -95,11 +95,11 @@
 
     private boolean setClear = false;
     private boolean captureLocation = false;
-    private Boolean foundFirstLocation = false;
-    private Boolean readOnly = false;
-    private Boolean draggable = false;
-    private Boolean intentDraggable = false;
-    private Boolean locationFromIntent = false;
+    private boolean foundFirstLocation = false;
```

Comment:
boolean fields default to false, so I suggest removing “ = false”. This will reduce the code volume, and, I argue, improve readability.

Reply:
That's right, I'll modify this code.
Thankyou @dcbriccetti